components/expenses/PayoutBankInformationForm.js

```diff
@@ -487,17 +487,13 @@ const PayoutBankInformationForm = ({ isNew, getFieldName, host, fixedCurrency, i
   const currencyFieldName = getFieldName('data.currency');
   const selectedCurrency = get(formik.values, currencyFieldName);
 
-  const validateCurrency = () => {
+  const validateCurrencyMinimumAmount = () => {
     // Skip if currency is fixed (2) (3)
     // or if `availableCurrencies` is not set (but we're not supposed to be there anyway)
     if (fixedCurrency || !availableCurrencies) {
       return;
     }
 
-    if (!formik?.values?.currency) {
-      return formatMessage({ defaultMessage: 'Please select a currency' });
-    }
-
```

Comment:
Could you explain why we're removing this?

Reply:
@znarf : This was added previously in https://github.com/opencollective/opencollective-frontend/pull/7530. For a multi-currency expense, after setting the amounts in the second step, when we go back to the first step and change the payout method we needed to reset the currency to null so that the user is forced to recheck the amounts in the new currency. In the first implementation I made the inputs null and also added the error message so that the user is shown this message. It looked like this, 

![image](https://user-images.githubusercontent.com/12435965/221065178-30154584-26e5-4933-8380-2168168c893d.png)

However later the implementation was changed to handle globally; https://github.com/opencollective/opencollective-frontend/pull/8584. So currently it looks like this,

![image](https://user-images.githubusercontent.com/12435965/221065045-68750c5c-888d-48b2-bb44-8b904f3306ba.png)

Note that we already have the "this field is required" so having this additional error I thought is not needed anymore.